docker-compose.yml

```diff
@@ -54,6 +54,7 @@ services:
       - ServiceRepositorySettings:ApiEndPointHost=altinn-repositories
       - ServiceRepositorySettings:RepositoryBaseURL=http://altinn-repositories:3000/
       - GeneralSettings:TemplateLocation=/Templates
+      - GeneralSettings:DeploymentLocation=/Deployment
```

Comment:
You need to set this environment variable in the `altinncore-deployment.yaml` file aswell?

Reply:
yeah, that seems right. Updated pr with changes